### ISSUE TITLE
[Script]: Spam Email Classification using LSTM

### ISSUE THREAD

**Issue by smty2018** [avinashkranjan/Amazing-Python-Scripts#2899]
### Aim

The model is trained to predict whether an email is spam or not based on the email text content.

### Details

The LSTM model architecture involves the following key components:

- `Embedding` Layer
- First `LSTM` Layer
- `Dropout` Layer.
- Second `LSTM` Layer
- `Dense` Layer

The model is trained using binary cross-entropy loss and the Adam optimizer.

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue
